src/run_perf_test.py

```diff
@@ -35,19 +36,29 @@ def main():
     parser.add_argument("--bundle-manifest", type=argparse.FileType("r"), help="Bundle Manifest file.", required=True)
     parser.add_argument("--stack", dest="stack", help="Stack name for performance test")
     parser.add_argument("--config", type=argparse.FileType("r"), help="Config file.", required=True)
+    parser.add_argument("--security", dest="security", action="store_true",
+                        help="Security of the cluster should be True/False",
+                        default=False)
     parser.add_argument("--keep", dest="keep", action="store_true", help="Do not delete the working temporary directory.")
     args = parser.parse_args()
 
     manifest = BundleManifest.from_file(args.bundle_manifest)
     config = yaml.safe_load(args.config)
 
+    tests_dir = os.path.join(os.getcwd(), "test-results", "perf-test")
+    os.makedirs(tests_dir, exist_ok=True)
```

Comment:
@gaiksaya can we use `test_recorder` here?

Reply:
Can be done in another PR. We have an issue opened for that here: https://github.com/opensearch-project/opensearch-build/issues/707